uobtheatre/bookings/serializers.py

```diff
@@ -164,10 +158,25 @@ def get_price_break_down(self, booking):
         return serialized_price_breakdown.data
 
 
-class CreateTicketSerializer(AppendIdSerializerMixin, serializers.ModelSerializer):
+class CreateTicketSerializer(serializers.ModelSerializer):
+    seat_group_id = serializers.PrimaryKeyRelatedField(
+        queryset=SeatGroup.objects.all(), source="seat_group", write_only=True
+    )  # TODO reduce queryset to only include those allowed for this performance
```

Comment:
I don't think this is possible without knowing the performance? The validation to avoid this is current in CreateBooking maybe the validation should happen in a validation method here?